src/navigation/NavigationTypes.ts

```diff
@@ -11,16 +11,8 @@ export type CountsType = {
   total: number
 }
 
-export interface Discipline {
-  id: number
-  title: string
-  numberOfChildren: number
-  isLeaf: boolean
-  apiKey?: string
-}
-
 interface ResultScreenData {
-  discipline: Discipline
+  discipline: DisciplineType
```

Comment:
Actually we planned to change to type/interface-naming without the `Type`-suffix, but as it is already inconsistent, we can do it later.